server/strong-oracle.js

```diff
@@ -0,0 +1,49 @@
+var settings = {};
```

Comment:
I cleaned up the duplicate code from the oracle test files. For memcache, I consolidated them into the same file, but the function calls for both are quite different, so Idk it is of much use. 
Since this is only a 2 point story, I did not spend too much time cleaning this up, but I'll add a story to make this app better, so we can all use it for internal testing